Composer/packages/lib/ui-shared/src/hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+/**
+ * Creates a textarea at then end of the document body then
+ * uses that to copy the specified text to the cliboard.
+ */
+
+export const copyToClipboard = (text: string) => {
+  // Remember the current selection
+  const rangeCount = document?.getSelection()?.rangeCount || 0;
+  const selected = rangeCount > 0 ? document?.getSelection()?.getRangeAt(0) : false;
```

Comment:
can document be undefined?

Reply:
No.  eslint coalesced to those operators for some reason.